### ISSUE TITLE
[1.18] Crushing shulker shells produces unexpected results

### ISSUE THREAD

**Issue by Snaitf** [SilentChaos512/Silent-Gear#463]
## Versions

- **Silent Gear**: 2.8.1
- **Silent's Gems**: 4.2.0 
- **Silent Lib**: 6.0.0
- **Forge**: 38.0.17
- **Modpack**: N/A 
- **Optifine Installed**: No 

## Expected Behavior

Crushing shulker shells in the crafting grid with a hammer should give one dust per shell.

## Actual Behavior

Crushing shulker shells in the crafting grid with a hammer yields unexpected results.  Sometimes I will get a dust, but the shell remains.  Sometimes the amount of shells in the grid actually increases by one, or even more.  My last 2 attempts was crushing 1 shell gave one dust, but left 2 shells in the grid.  then taking another dust increased my amount of shells to 5.  I first noticed something was wrong when i put 6 shells in the grid with my hammer, and shift-clicked the dust out, and got over 50 dust from the 6 shells.

## Links/Images

- N/A

## Steps to Reproduce the Problem

1.  Place shulker shells and hammer in the crafting grid.
2.  Remove the crushed shulker shell output.
3.  Notice the odd behaviour.

**Comment by CaribooWho** [comment]
Placing multiple shulker shells in a stack in the crafting grid allows for duplication. Starting with 2 yields 3 after crafting, starting with 5 yields 9, 15 yields 29, etc. Essentially the amount of shells you can expect from a given input is 2n-1, which explains why starting with 1 shell only duplicates the powder and does not increase the number of shells in the crafting grid.

**Comment by SilentChaos512** [comment]
I wish I could actually reproduce this issue. I have no way to test it, so I can't figure out where the bug is. Perhaps I just need to change these recipes to use a custom crafting table, since I believe this issue is caused by other mods.

**Comment by CaribooWho** [comment]
yes i am playing in the pack Team Rustic 4, maybe it's an inventory mod like mouse/inventory tweaks or something?

**Comment by Bultjer** [comment]
> I wish I could actually reproduce this issue. I have no way to test it, so I can't figure out where the bug is. Perhaps I just need to change these recipes to use a custom crafting table, since I believe this issue is caused by other mods.

I have the issue currently in the pack atm7.
Iron hammer out of 6 iron ingots and a wooden tool rod.
It is happening in the crafting grid from refined storage, but also just in the player inventory.
The hammer and any stack size of shulker shells in the crafting grid. When taking out one crushed shell piece, one shell gets used but it spawns in the slot of the stack the initial stack size, even past 64 in one slot.
So far I only have found shulker shells doing this.

ATM7 0.4.23
Silent Gear 1.18.2-2.10.10

**Comment by lele92007** [comment]
noticed a similar issue with silent-gear-1.16.5-2.6.30
the shulker shell stayed in the crafting grid
I only tested in applied, but the same thing probably happens in inventory

**Comment by Mitchell5200** [comment]
ATM pack team member here, i've managed to trace this to the Supplementaries mod, it also interacts with shulker shells for some reason, and changing `shulker_shell` to false in the config `supplementaries-registry.toml` fixes this issue. will be fixed in the next ATM7 pack release of 0.4.33
